daprdocs/content/en/reference/arguments-annotations-overview.md

```diff
@@ -36,6 +36,8 @@ This table is meant to help users understand the equivalent options for running
 | `--metrics-port` | `--metrics-port` | | `dapr.io/metrics-port` | Sets the port for the sidecar metrics server. Default is `9090` |
 | `--mode` | not supported | | not supported | Runtime hosting option mode for Dapr, either `"standalone"` or `"kubernetes"` (default `"standalone"`). [Learn more.]({{< ref hosting >}}) |
 | `--placement-host-address` | `--placement-host-address` | | `dapr.io/placement-host-address` | Comma separated list of addresses for Dapr Actor Placement servers. When no annotation is set, the default value is set by the Sidecar Injector. When the annotation is set and the value is empty, the sidecar does not connect to Placement server. This can be used when there are no actors running in the sidecar. When the annotation is set and the value is not empty, the sidecar connects to the configured address. For example: `127.0.0.1:50057,127.0.0.1:50058` |
+| `--actors-service` | not supported | | not supported | Configuration for the service that offers actor placement information. The format is `<name>:<address>`. For example, setting this value to `placement:127.0.0.1:50057,127.0.0.1:50058` is an alternative to using the `--placement-host-address` flag. |
```

Comment:
Shouldn't these be called "--actors-service-address" and "--reminder-service-address" as per the code here
https://github.com/DeepanshuA/dapr/blob/6591003f8890b2a67fc51816faf7aecf90d49824/charts/dapr/charts/dapr_sidecar_injector/templates/dapr_sidecar_injector_deployment.yaml#L173
and to keep this consistent with the `--placement-host-address` naming convention?

Reply:
I think `actors-service`/`reminders-service` also seek to get the name of the service, along with the address. Whereas, `placement-host-address` was just asking particularly for address.

So, to make it generic for name and address combined, `actors-service` or `reminders-service` also sounds fine to me.